go.mod

```diff
@@ -207,4 +207,4 @@ require (
 
 replace github.com/ethereum-optimism/optimism v1.7.2 => ./
 
-replace github.com/ethereum/go-ethereum v1.13.8 => github.com/kroma-network/go-ethereum v0.5.0-rc.4
+replace github.com/ethereum/go-ethereum v1.13.8 => github.com/kroma-network/go-ethereum v0.5.0
```

Comment:
Please update docker compose file.

Reply:
Oh, sorry about that. I missed this comment. I will do it in the following PR.